src/api/src/main.py

```diff
@@ -26,11 +33,14 @@ def home():
 
 @app.get("/download/{story_id}")
 async def download_book(
-    story_id: int,
+    story_id: str,
```

Comment:
Let's make this `story_url` and add another `story_id` parameter. Only one is required.

Story URL would be reflected on the front end, and story id is present for easier integration with code and for backward compatibility.

Reply:
Oops, scratch this!

Please refer to https://github.com/TheOnlyWayUp/WattpadDownloader/pull/8#discussion_r1667403451